app/migrations/9999_postgis.py

```diff
@@ -0,0 +1,9 @@
+from django.db import migrations
+
+class Migration(migrations.Migration):
+    dependencies = [("app", "0001_initial")]
+
+    operations = [
+        migrations.RunSQL("SET postgis.enable_outdb_rasters TO True;"),
+        migrations.RunSQL("SET postgis.gdal_enabled_drivers TO 'GTiff';")
```

Comment:
Good catch, it actually is not persistent, I replaced it with a ALTER SYSTEM SET ... Thanks!!